src/reflect/scala/reflect/internal/Variances.scala

```diff
@@ -300,17 +301,21 @@ trait Variances {
       case MethodType(params, restpe)                      => inSyms(params).flip  & inType(restpe)
       case PolyType(tparams, restpe)                       => inSyms(tparams).flip & inType(restpe)
       case ExistentialType(tparams, restpe)                => inSyms(tparams)      & inType(restpe)
-      case AnnotatedType(annots, tp)                       => inAnnots(annots)     & inType(tp)
+      case AnnotatedType(annots, tp) =>
+        if (considerUnchecked && annots.exists(_.matches(definitions.uncheckedVarianceClass))) Bivariant
+        else inAnnots(annots) & inType(tp)
```

Comment:
The vertically-aligned version!

```scala
      case AnnotatedType(annots, _) if unchecked(annots)   => Bivariant
      case AnnotatedType(annots, tp)                       => inAnnots(annots)     & inType(tp)
```

😄

Reply:
We could make it happen 😄